ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -294,6 +296,7 @@ private class JdbcLedgerDao(
       offsetStep: OffsetStep,
       partyEntry: PartyLedgerEntry,
   )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing party entry")
```

Comment:
PartyEntry contains submissionId, please log it as well

Reply:
Re: this and other comments.

Before acting on this you may want to check whether these are already injected in the logging context upstream.

See `ExecuteUpdate#loggingContextFor`

```scala
      case PartyAllocationRejected(submissionId, participantId, _, rejectionReason) =>
        Map(
          "updateSubmissionId" -> submissionId,
          "updateParticipantId" -> participantId,
          "updateRejectionReason" -> rejectionReason,
        )
      case PublicPackageUpload(_, sourceDescription, _, submissionId) =>
        Map(
          "updateSubmissionId" -> submissionId.getOrElse(""),
          "updateSourceDescription" -> sourceDescription.getOrElse(""),
        )
```

Reply:
The `ExecuteUpdate#loggingContextFor` method enriches logging context for data writes in `JdbcLedgerDao`. There's no need to enrich the logging context in the latter.
However, I think we should consider removing `update` prefix from the context keys to be consistent with other places where we log correlation ids (`commandId`, `transactionId`, `submissionId`). What do you think @stefanobaghino-da @mziolekda ?

Reply:
I am strongly for removing that prefix. It leaks a bit of architecture detail to the log (updateCommandId differs from commandId only by the place in the code that prints them). For an end-user or support personnel it can be confusing.

Reply:
Also, the chain of logging context must be broken somewhere, because the example you posted elsewhere did not contain a context:
```
INFO: Storing package entry
```

Reply:
From what I see in the codebase, there's no broken chain of logging context but rather the same method for storing package entries is used in different places. In some of them the logging context is not used at all. However, for the cases pertinent to this PR, the logging context is preserved.

Reply:
> I am strongly for removing that prefix. It leaks a bit of architecture detail to the log (updateCommandId differs from commandId only by the place in the code that prints them). For an end-user or support personnel it can be confusing.

I've removed the prefix. The `LoggingContext`'s keys should now have uniform naming.

Reply:
Let's have a look at the example log-line from an actual run and close this thread if the context appears

Reply:
An example log line for this case is:
```
2021-02-10T18:07:01.537Z INFO  c.d.p.s.d.JdbcLedgerDao@[kka.actor.default-dispatcher-6] -
                Storing package entry (context: {participantId=sandbox-participant})
```